admin_manual/release_schedule.rst

```diff
@@ -7,6 +7,8 @@ This page gives an overview of the currently supported and released versions as
 +--------+-------------------+------------------+--------------------------+---------------------------+
 |        | release date      | end of life      | latest version           | next version              |
 +========+===================+==================+==========================+===========================+
+| **17** | 2019-08-08        | 2020-08          |                       -  | *17.0.0 (2019-08-08)*     |
++--------+-------------------+------------------+--------------------------+---------------------------+
```

Comment:
This was on purpose - we don't want to put major release dates into the manual, as this is a document that is also shipped in the release and thus easily out of date.

Reply:
> This was on purpose - we don't want to put major release dates into the manual, as this is a document that is also shipped in the release and thus easily out of date.

Ah ok. didn't know that.